.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -53,3 +53,15 @@ body:
       options:
         - label: Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
           required: true
+
+  - type: dropdown
+    id: priority
+    attributes:
+      label: Priority
+      description: What is this feature's priority from your product's perspective
+      options:
+        - Low
+        - High
+        - Critical
```

Comment:
Should we align with the requested priority on the bug template?

![image](https://github.com/microsoft/fluentui/assets/40304143/957549a0-018b-4f81-bcee-a5b84fd4666a)

Reply:
To me the "Blocking" value makes sense for issues, but not necessarily for features as they are usually planned and communicated beforehand. I'm ok with changing the other values.

Reply:
sounds good, I agree, having the same low/normal/high is good